templates/gift_card.liquid

```diff
@@ -135,6 +139,7 @@
             alt=""
             height="{{ 570 | divided_by: 1.5 }}"
             width="570"
+            loading="eager"
```

Comment:
Good point! I think we can still keep this for now because `eager` is the default value anyway. Once Theme Check version is bumped, we could go through the codebase and choose to remove all instances of `loading="eager"` if we want.